cmd/atlas-action/main.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import (
+	"context"
+	"fmt"
+
+	"ariga.io/atlas-go-sdk/atlasexec"
+	"github.com/alecthomas/kong"
+	"github.com/sethvargo/go-githubactions"
+)
+
+const (
+	CmdMigratePush  = "migrate/push"
+	CmdMigrateLint  = "migrate/lint"
+	CmdMigrateApply = "migrate/apply"
+)
+
+var cli RunAction
+
+func main() {
+	action := githubactions.New()
+	c, err := atlasexec.NewClient("", "atlas")
+	if err != nil {
+		action.Fatalf("Failed to create client: %s", err)
+	}
+	cli := kong.Parse(&cli, kong.Bind(context.Background(), c, action))
+	if err := cli.Run(); err != nil {
+		action.Fatalf("Failed to run command: %s", err)
+	}
+}
+
+// RunAction is a command to run one of the Atlas GitHub Actions.
+type RunAction struct {
+	Action string `help:"Command to run" required:""`
+}
+
+func (r *RunAction) Run(_ context.Context, _ *atlasexec.Client, _ *githubactions.Action) error {
```

Comment:
Another way it to let packages register themselves and empty import.

Reply:
Just an idea, not a suggestion

Reply:
When this gets large enough , initially we will have 3-4 opts so no big deal